.solhint.json

```diff
@@ -1,18 +1,13 @@
 {
   "extends": "solhint:recommended",
   "rules": {
```

Comment:
I vote for warning on [`named-parameters-mapping`](https://github.com/solhint-community/solhint-community/blob/master/docs/rules/naming/named-parameters-mapping.md) rule. A shout-out to @0xteddybear for noticing it.